GameWorld/Configurations/InputConfigManager.cs

```diff
@@ -0,0 +1,189 @@
+            AddDescriptionActionKeyBinding("Camera rotate right", ActionType.CameraRotateRight, keyBindingList);
+
+            keyBindingList = new List<KeyBinding>() { new KeyBinding(ControllerType.Keyboard, Keys.Up) };
+            AddDescriptionActionKeyBinding("Camera rotate up", ActionType.CameraRotateUp, keyBindingList);
+
+            keyBindingList = new List<KeyBinding>() { new KeyBinding(ControllerType.Keyboard, Keys.Down) };
+            AddDescriptionActionKeyBinding("Camera rotate down", ActionType.CameraRotateDown, keyBindingList);
+
+            keyBindingList = new List<KeyBinding>() { new KeyBinding(ControllerType.Keyboard, Keys.P) };
+            AddDescriptionActionKeyBinding("Toggle wireframe", ActionType.ToggleWireframe, keyBindingList);
+
+            keyBindingList = new List<KeyBinding>() { new KeyBinding(ControllerType.Keyboard, Keys.T) };
+            AddDescriptionActionKeyBinding("Toggle fullscreen", ActionType.ToggleFullscreen, keyBindingList);
+
+            keyBindingList = new List<KeyBinding>() { new KeyBinding(ControllerType.Keyboard, Keys.R) };
+            AddDescriptionActionKeyBinding("Toggle fixed framerate", ActionType.ToggleFixedFramerate, keyBindingList);
+
+            keyBindingList = new List<KeyBinding>() { new KeyBinding(ControllerType.Keyboard, Keys.Escape) };
+            AddDescriptionActionKeyBinding("Exit", ActionType.Exit, keyBindingList);
+        }
+
+        private static void AddDescriptionActionKeyBinding(string description, ActionType actionType, List<KeyBinding> keyBindingList)
+        {
+            descriptionToActionBinding.Add(description, actionType);
+            actionToKeyBindings.Add(actionType, keyBindingList);
+        }
+
+        public static string AssignKeyToAction(string description, List<KeyBinding> keyBindingList, bool isForceAssign)
+        {
+            ActionType actionType = descriptionToActionBinding[description];
+
+            if (isForceAssign == false)
+            {
+                foreach (var actionToKeyPair in actionToKeyBindings)
+                {
+                    if ((actionToKeyPair.Value == keyBindingList) && (actionToKeyPair.Key != actionType))
+                    {
+                        foreach (var descriptionToActionPair in descriptionToActionBinding)
+                        {
+                            if (descriptionToActionPair.Value == actionToKeyPair.Key)
+                            {
+                                return descriptionToActionPair.Key;
+                            }
+                        }
+                    }
+                }
+            }
+            else
+            {
+                actionToKeyBindings[actionType] = keyBindingList;
+            }
+
+            return "";
+        }
+
+        public static bool IsActionKeysDown(ActionType actionType)
+        {
+            List<KeyBinding> keyBindingList = actionToKeyBindings[actionType];
+            bool result = true;
```

Comment:
Because it became false only if released key found. If not, it remains and returned as true.